doc/move-vat-to-existing-leu_plantuml.txt

```diff
@@ -0,0 +1,92 @@
+@startuml
```

Comment:
It would be useful to have a link in the `README` on the UML tool you are using.

Reply:
Have added a separate README to the doc file containing a link to PlantUML.